olm/strimzi-cluster-operator.clusterserviceversion.yaml

```diff
@@ -525,32 +531,39 @@ spec:
   customresourcedefinitions:
     owned:
     - name: kafkas.kafka.strimzi.io
-      version: v1alpha1
+      version: v1beta1
       kind: Kafka
       displayName: Kafka
       description: Represents a Kafka cluster
+      statusDescriptors:
```

Comment:
Wait ... isn't this missing some part? It has to point to the conditions. You should probably also copy the other parts missing here.